iocore/net/quic/QUICLossDetector.cc

```diff
@@ -242,14 +243,14 @@ QUICLossDetector::_on_ack_received(const QUICAckFrame &ack_frame)
     return;
   }
 
+  this->_detect_lost_packets();
+
   this->_crypto_count = 0;
   this->_pto_count    = 0;
 
   QUICLDVDebug("Unacked packets %lu (retransmittable %u, includes %u handshake packets)", this->_sent_packets.size(),
```

Comment:
Printing the same thing without doing nothing doesn't make sense :) Let's move the first one too.